File: Pattern/pattern7/pattern.js
const pattern = (n) => {
  for (let i = 0; i <= n; i++) {
    let line = "";
    for (let j = 1; j <= n - i; j++) {
      line += " ";
    }
    for (let k = 1; k <= i; k++) {
      line += " *";
    }
    console.log(line);
  }
};

pattern(8)

File: Pattern/pattern7/pattern.go
package main

import "fmt"

func pattern(x int) {
	for i := 0; i < x; i++ {
		for j := 0; j <= x-i; j++ {
			fmt.Print(" ")
		}

		for k := 0; k <= i; k++ {
			fmt.Print(" *")
		}

		fmt.Println()
	}
}

func main() {
	pattern(3)
}
